mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -44,11 +49,17 @@
         private final MailAddress sender;
         private final Collection<MailAddress> recipients;
         private final MimeMessage msg;
+        private final List<String> attributeNames;
 
-        public SentMail(MailAddress sender, Collection<MailAddress> recipients, MimeMessage msg) {
+        public SentMail(MailAddress sender, Collection<MailAddress> recipients, MimeMessage msg, List<String> attributeNames) {
             this.sender = sender;
-            this.recipients = recipients;
+            this.recipients = ImmutableList.copyOf(recipients);
             this.msg = msg;
+            this.attributeNames = Lists.newArrayList(attributeNames);
```

Comment:
No, that is the point of this changeset : we care : we want to assert it is empty !